plugins/auth-basic.yml

```diff
@@ -1,7 +1,7 @@
 id: auth-basic
 name: Basic Auth Provider
 description: Add basic (username-password) authentication and authorization to your service. This plugin must be installed with the “NestJS Auth Module” plugin.
-repo: amplication/plugins
+repo: https://github.com/amplication/plugins/tree/main/plugins/auth-basic
```

Comment:
what is the tree and main folder ? is it new ?

Reply:
it is there for all the others plugin, we might need to default it to the root level, right?